src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
Takin into account that the "dual" metho is setDbPrefix() this should be getDbPrefix().

In addition, for homogenity, given that setDbPrefix() uses a std::string as argument, getDbPrefix() should return a std::string also.